Flank/e2e_test.sh

```diff
@@ -9,4 +9,4 @@ APP_APK=$APK_ROOT/app-debug.apk
 TEST_APK=$APK_ROOT/app-debug-androidTest.apk
 
 set -x
-java -jar ./build/libs/Flank-*.jar $APP_APK $TEST_APK
+java -jar ./build/libs/Flank-*.jar -a $APP_APK -t $TEST_APK -f "class com.example.app.ExampleUiTest#testHelloWorld"
```

Comment:
oh I didn't know this existed. how come this is not part of the ci?

Reply:
Not sure, maybe @renas could help here. I use that script to test my changes locally.